File: src/primus/pawel/Exercise_076.java
package primus.pawel;

import java.util.Arrays;
import java.util.LinkedList;
import java.util.List;

// Testing 1-2-3
// 7kyu
// https://www.codewars.com/kata/54bf85e3d5b56c7a05000cf9/train/java
// 2021-04-14 T:19:01:50
class Exercise_076 {

    public static List<String> number(List<String> lines) {
        List<String> resultList = new LinkedList<>();
        for (int i = 0; i < lines.size(); i++) {
            resultList.add(i+1 + ": " + lines.get(i));
        }
        return resultList;
    }

    public static void main(String[] args) {
        number(Arrays.asList("a", "b", "c"));
    }


}

File: src/JS/Exercise_076.js

// No zeros for heros
// 8 kyu
// https://www.codewars.com/kata/570a6a46455d08ff8d001002/train/javascript
// 2022-05-28 T:12:31:58

function noBoringZeros(n) {
    return Number(n.toString().replace(/^0+(\d)|(\d)0+$/gm, '$1$2'));
}

console.log(noBoringZeros(13000))